src/main/java/uk/gov/hmcts/reform/pip/account/management/service/MediaLegalApplicationService.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.hmcts.reform.pip.account.management.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.pip.account.management.database.AzureBlobService;
+import uk.gov.hmcts.reform.pip.account.management.database.MediaLegalApplicationRepository;
+import uk.gov.hmcts.reform.pip.account.management.model.MediaAndLegalApplication;
+import uk.gov.hmcts.reform.pip.account.management.model.MediaLegalApplicationStatus;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class MediaLegalApplicationService {
+
+    private final MediaLegalApplicationRepository mediaLegalApplicationRepository;
+
+    private final AzureBlobService azureBlobService;
+
+    LocalDateTime now = LocalDateTime.now();
+
+    @Autowired
+    public MediaLegalApplicationService(MediaLegalApplicationRepository mediaLegalApplicationRepository,
+                                        AzureBlobService azureBlobService) {
+        this.mediaLegalApplicationRepository = mediaLegalApplicationRepository;
+        this.azureBlobService = azureBlobService;
+    }
+
+    /**
+     * Get a list of all the applications.
+     *
+     * @return A list of all applications
+     */
+    public List<MediaAndLegalApplication> getApplications() {
+        return mediaLegalApplicationRepository.findAll();
+    }
+
+    /**
+     * Get a list of applications by the status.
+     *
+     * @param status The MediaLegalApplicationStatus enum to retrieve applications by
+     * @return A list of all applications with the relevant status
+     */
+    public List<MediaAndLegalApplication> getApplicationsByStatus(MediaLegalApplicationStatus status) {
+        return mediaLegalApplicationRepository.findByStatus(status);
+    }
+
+    public MediaAndLegalApplication getApplicationById(UUID id) {
+        return mediaLegalApplicationRepository.findById(id).get();
```

Comment:
none of the searches here handle error states, 
this findbyId returns an optional, your .get will return if its found but you dont handle a id not found